src/screens/profile/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useRef } from 'react';
+import { Dimensions, ScrollView, View } from 'react-native';
+
+import { MainButton, Text } from 'src/components';
+import { AppColor, ButtonMode } from 'src/common/enums';
+
+import styles from './styles';
+
+type State = Array<{ id: number; text: string }>;
+
+interface Props {
+  active: number;
+  handleClick: (a: number) => void;
+}
+
+const WIDTH = Dimensions.get('screen').width * 0.8;
+const BUTTON_WIDTH = Math.ceil(WIDTH / 3);
+
+const Navbar = ({ active, handleClick }: Props) => {
+  const scrollRef = useRef<ScrollView>(null);
+
+  scrollRef.current?.scrollTo({
+    x: active * BUTTON_WIDTH,
+    y: 0,
+    animated: true,
+  });
+
+  const [items] = React.useState<State>([
```

Comment:
Just `useState<State>` (and import this function like this: `import React, { useState, useEffect } from 'react'`)